### ISSUE TITLE
relay station

### ISSUE THREAD

**Issue by amir-arad** [starwards/starwards#1211]
- [ ] state of each navigation waypoint set: shown/hidden
 - [ ] state for course waypoint selection
widgets:
 - [ ] dradis radar ( blocked by #1209 )
 - [ ] navigation waypoints layers toggle state ( blocked by #1210 )

hotkeys: 
- [ ] toggle each navigation waypoint layer
- [ ] toggle on/off waypoint creation mode
- [ ] delete selected waypoint(s)

TBD probes

**Comment by amir-arad** [comment]
Deferred to post-first-event per "ship early, expand later" strategy. First LARP event will run a 4-station bridge (Pilot, Weapons, ECR, Signals). Relay is Tier 3.

Also note: blockers #1185 (waypoints) and #1214's blocker are now closed — downstream issues can be unblocked.

See starwards-design/roadmap.md and starwards-design/decisions/004-ship-early-expand-later.md.

**Comment by amir-arad** [comment]
Backlog triage (2026-07-19): moved to the design KB — https://github.com/starwards/starwards-design/blob/main/product/backlog.md#navigation--expansion-stations-post-first-event-per-ship-early-decision

GitHub issues are now dispatch tickets only ([ADR-0003](https://github.com/starwards/starwards-design/blob/main/governance/decisions/0003-github-as-dispatch-only.md)). To pick this up again, open a fresh dispatch issue from the backlog entry (don't reopen this one).
